uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/apis/work-pattern-api.js

```diff
@@ -10,18 +11,19 @@ define([
     return api.extend({
 
       /**
-       * This method returns the calendar for a specific period, as a list of days and their type
+       * This method returns the calendar for the given contact(s) and period,
+       * as a list of days and their type
        *
-       * @param {string} contactId The ID of the Contact
-       * @param {string} periodId The ID of the Absence Period
-       * @param {object} params
+       * @param {string/int/Array} contactId can be also an array for multiple contacts
```

Comment:
honestly, i don't know which is it, do you know? Also the fact it can be (potentially) either string or int it's not important here, since we are not making any `===` comparison